server/validation/userAuth.js

```diff
@@ -0,0 +1,63 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+
+const userAuthHelper = {
+  /**
+   * Hash Password Method
+   * @param {string} password
+   * @returns {string} returns hashed password
+   */
+  hashPassword(password) {
+    return bcrypt.hashSync(password, bcrypt.genSaltSync(8));
+  },
+  /**
```

Comment:
Please leave a new line before JSDocs
```suggestion

  /**
```